src/Login.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import {
+  Grid,
+  Row,
+  Col,
+  FormGroup,
+  ControlLabel,
+  FormControl,
+  HelpBlock,
+  Button,
+  Alert
+} from 'react-bootstrap'
+import { withRouter } from 'react-router'
+
+import NavBar from './components/NavBar'
+import { Api } from './utils'
+
+function FieldGroup({ id, label, help, ...props }) {
+  return (
+    <FormGroup controlId={id}>
+      <ControlLabel>{label}</ControlLabel>
+      <FormControl {...props} />
+      {help && <HelpBlock>{help}</HelpBlock>}
+    </FormGroup>
+  )
+}
+
+class Login extends Component {
+  state = {
+    error: null
+  }
+
+  onSubmitForm = e => {
+    e.preventDefault()
+    const { username, password } = e.target
+    const data = {
+      username: username.value,
+      password: password.value
+    }
+
+    // TODO: replace URL with correct url
+    Api.postLogin(data)
+      .then(({ message: error, status }) => {
+        this.props.history.push('/', { isEditing: true })
+        if (status === 'failed') {
+          this.setState({ error })
+        } else {
+          this.props.history.push('/', { isEditing: true })
+        }
+      })
+      .catch(err => {
+        //TODO: Handle any error from request
+        console.error(err)
+        this.setState({ error: 'An unkonwn error occured' })
+      })
+  }
+
+  render() {
+    return (
+      <div>
+        <NavBar isEditing={false} />
+        <Grid>
+          <Row>
+            <Col md={4} xs={18} mdOffset={4}>
+              <form onSubmit={this.onSubmitForm}>
+                {' '}
```

Comment:
Looks like you're trying to space things using html, which is not what html should be used for. Instead we should use css but we don't need to worry about styling yet